suite-common/wallet-core/src/transactions/transactionReducer.ts

```diff
@@ -0,0 +1,145 @@
+import { accountsActions } from 'suite-common/wallet-core/src/index';
+
+import { Account, WalletAccountTransaction } from '@suite-common/wallet-types/libDev/src';
```

Comment:
imports from `libDev/` 😬 that definitely should not happen. We need to ban this in eslint.